services/device-registry-base/src/main/java/org/eclipse/hono/service/management/credentials/CommonSecret.java

```diff
@@ -160,4 +160,7 @@ public void checkValidity() {
             }
         }
     }
+
+    void merge(final CommonSecret secret) {
```

Comment:
I think that this method deserves a JavaDoc comment, doesn't it?

Reply:
I have now added the JavaDoc.